rootfs/healthcheck.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Get latest log file
+PFCLIENT_LOG_FILE=$(find /var/log/pfclient -type f -iname "pfclient-log_*.log" | sort | tail -1)
```

Comment:
It looks like this healthcheck will need to be re-written for ADSBHub. I'd suggest removing it for now and we can tackle writing a healthcheck once the container is working.